File: AgendaUniversitaria/src/com/untref/au/agendauniversitaria/model/Documento.java
package com.untref.au.agendauniversitaria.model;

public class Documento {

	private String mCodigo;
	private String mNombre;
	private String mLink;
	private Materia mMateria;

	public String getmCodigo() {
		return mCodigo;
	}

	public void setmCodigo(String mCodigo) {
		this.mCodigo = mCodigo;
	}

	public String getmNombre() {
		return mNombre;
	}

	public void setmNombre(String mNombre) {
		this.mNombre = mNombre;
	}

	public String getmLink() {
		return mLink;
	}

	public void setmLink(String mLink) {
		this.mLink = mLink;
	}

	public Materia getmMateria() {
		return mMateria;
	}

	public void setmMateria(Materia mMateria) {
		this.mMateria = mMateria;
	}

}

File: AgendaUniverstaria-Api/ClasesAgendaUniversitaria/ClasesAgendaUniversitaria/Documento.vb
Public Class Documento

    Private mCodigo As String
    Public Property Codigo() As String
        Get
            Return mCodigo
        End Get
        Set(ByVal value As String)
            mCodigo = value
        End Set
    End Property


    Private mNombre As String
    Public Property Nombre() As String
        Get
            Return mNombre
        End Get
        Set(ByVal value As String)
            mNombre = value
        End Set
    End Property


    Private mLink As String
    Public Property Link() As String
        Get
            Return mLink
        End Get
        Set(ByVal value As String)
            mLink = value
        End Set
    End Property


    Private mMateria As Materia
    Public Property Materia() As Materia
        Get
            Return mMateria
        End Get
        Set(ByVal value As Materia)
            mMateria = value
        End Set
    End Property



End Class
